packages/contracts/test/test-utils/typechain-versions.ts

```diff
@@ -0,0 +1,9 @@
+/// Typechain will sometimes by default link to the wrong version of the contract, when we have name collisions
+/// The version specified in src is the factory and contract without the version number.
+/// Import as needed in the test files, and use the correct version of the contract.
+
+export {Multisig__factory as Multisig_V1_0_0__factory} from '../../typechain/factories/@aragon/osx-v1.0.0/plugins/governance/multisig/Multisig__factory';
+export {Multisig__factory as Multisig_V1_3_0__factory} from '../../typechain/factories/@aragon/osx-v1.3.0/plugins/governance/multisig/Multisig__factory';
+export {Multisig__factory} from '../../typechain/factories/src/Multisig__factory';
+export {Multisig} from '../../typechain/src/Multisig';
+export {IMultisig} from '../../typechain/src/IMultisig';
```

Comment:
We have to understand why this is required

Reply:
If you try and import from the `index.ts` file at the root of the autogenerated `typechain` folder, then you will see that, when we import multiple versions of the same named contract, typechain doesn't export the correct one (at least in my case with the tokenVoting). Specifying a particular version in this way ensures you use the correct version of the contracts at all times.